crates/miden-objects/src/account/code/mod.rs

```diff
@@ -19,38 +19,40 @@ use crate::Word;
 use crate::account::{AccountComponent, AccountType};
 
 pub mod procedure;
-use procedure::{AccountProcedureInfo, PrintableProcedure};
+use procedure::{AccountProcedureRoot, PrintableProcedure};
 
 // ACCOUNT CODE
 // ================================================================================================
 
-/// A public interface of an account.
+/// The public interface of an account.
 ///
-/// Account's public interface consists of a set of account procedures, each procedure being a
-/// Miden VM program. Thus, MAST root of each procedure commits to the underlying program.
+/// An account's public interface consists of a set of account procedures, each of which is
+/// identified and committed to by a MAST root. They are represented by [`AccountProcedureRoot`].
 ///
-/// Each exported procedure is associated with a storage offset and a storage size.
-///
-/// We commit to the entire account interface by building a sequential hash of all procedure MAST
-/// roots and associated storage_offset's. Specifically, each procedure contributes exactly 8 field
-/// elements to the sequence of elements to be hashed. These elements are defined as follows:
+/// The code commits to the entire account interface by building a sequential hash of all procedure
+/// MAST roots. Specifically, each procedure contributes exactly 4 field elements to the sequence of
+/// elements to be hashed. Each procedure is represented by its MAST root:
```

Comment:
I think we should probably mention somewhere here that procedures are not sorted by anything. Or maybe that the order of procedures could be defined arbitrarily at account creation time.

Maybe also worth mentioning is that the first procedure in the list will be the authentication procedure.